README.md

```diff
@@ -1,10 +1,13 @@
 # Authorization Code Flow + PKCE DotNet Sample and Test
 
+| :loudspeaker: **Notice**: Samples have been updated to reflect that they work on AVEVA Data Hub. The samples also work on OSIsoft Cloud Services unless otherwise noted. |
+| -----------------------------------------------------------------------------------------------|  
+
 **Version:** 1.2.23
```

Comment:
This one i would bump the version since we are renaming functions.  Probably 1.3.0

Reply:
And history file

Reply:
great point, I will do that for the others that follow this behavior as well. Thanks!